### PR TITLE
feat(skymp5-server): print unhandled messages

### PR DESCRIPTION
**Pull request by artemmartyhin**

### CODE REVIEW

File: skymp5-server/cpp/server_guest_lib/ActionListener.cpp [hunk 1]
```diff
@@ -725,3 +725,9 @@ void ActionListener::OnHit(const RawMessageData& rawMsgData_,
                 hitData.target, damage, currentHealthPercentage,
                 healthPercentage);
 }
+
+void ActionListener::OnUnknown(const RawMessageData& rawMsgData,
+                               simdjson::dom::element data)
+{
+  spdlog::debug("Got unhandled message: {}", data);
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/ActionListener.cpp:732]: data.get_c_str() intstead

File: skymp5-client/src/remoteServer.ts [hunk 2]
```diff
@@ -142,6 +142,7 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
```
Comment (Pospelove) [inline skymp5-client/src/remoteServer.ts:142]: please remove ts changes

**Review by Pospelove — CHANGES_REQUESTED** [review]

**Review by Pospelove — APPROVED** [review]

**Comment by Pospelove** [comment]
oops approved by mistake

**Comment by Pospelove** [comment]
@nic11 Use https://simdjson.org/api/0.3.1/classsimdjson_1_1minify.html
get_string seem to get string from string value instead of printing as json

**Comment by nic11** [comment]
Gotta test it somehow...

**Review by Pospelove — APPROVED** [review]

**Review by Pospelove — APPROVED** [review]

**Pull request merged**